packages/mdc-floating-label/_mixins.scss

```diff
@@ -68,6 +68,38 @@
   }
 }
 
+@mixin mdc-floating-label-outlined-float-position($positionY, $positionX: 0%, $scale: .75) {
```

Comment:
Is there some way we could fathom integrating this into the notched-outline package rather than adding a notched-outline-specific mixin to floating-label? (e.g. would it be feasible to invoke the `mdc-floating-label-float-position` mixin twice within a mixin intended to set up the floating label within notched-outline?)

Otherwise, this seems like it might be confusing to use externally.

Reply:
Yes, updated.